### ISSUE TITLE
Mod interfering with Better Archery

### ISSUE THREAD

**Issue by eppersonadam** [Vapok/AdventureBackpacks#137]
**Describe the bug**

[Error  :vapok.mods.adventurebackpacks] [AdventureBackpacks.Patches.PlayerPatches+PlayerHaveRequirementItemsPatch+<Transpiler>d__0.MoveNext] HaveRequirementItems Transpiler Failed To Patch

Quiver slots Fail to show up.

**To Reproduce**
1. Go to '...' Run Dedicated server w/ Better Archery

**Expected behavior**

Mod works with single player, just not on dedicated server.

**Screenshots**

![image](https://github.com/user-attachments/assets/c543035b-3d47-4988-9cf6-b6e683eb6d5f)

**Mod List (please list your mods that are being used):**
 - Provide a list or attach your Player Log - Running fine on Player Log, Dedicated Server issue only.
 - Code for List of Mods:
 - 01931420-01f1-5a57-f8ff-8149ac5b4780

**Additional context**

**Comment by Vapok** [comment]
HI there, I would need to see the full player log.

What this is likely telling me is that another mod is overwriting the same transpiler that I am, which may introduce an incompatibility.  But to figure it out, I'll need the full log.

**Comment by eppersonadam** [comment]
[Player.log](https://github.com/user-attachments/files/17693025/Player.log)
[LogOutput.log](https://github.com/user-attachments/files/17693026/LogOutput.log)

**Comment by Vapok** [comment]
This is helpful.

So looks like Better Archery is changing some code with a transpiler in the same area that I am trying to.

I'll have to spend some time with BA to figure out what it's doing.

I assume Adventure Backpacks works fine with our Better Archy?

**Comment by eppersonadam** [comment]
Yes, it works fine without BA. Everything works on a single player game not issues, just with dedicated. Thanks though!

**Comment by Caledor2000** [comment]
I don't think Better Archery does anything when installed on the server. I run my dedicated server with Adventure Backpacks on the server and clients, and Better Archery on the clients only, and it works fine.

**Comment by eppersonadam** [comment]
okay, i'll possibly try to remove it from server and run only client side
